ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -8187,6 +8188,27 @@ static void unwindAddFiles(HqlExprArray & args, IHqlExpression * expr, bool isOr
         args.append(*LINK(expr));
 }
 
+static IHqlExpression * queryRootConcatActivity(IHqlExpression * expr)
+{
+    loop
+    {
+        node_operator curOp = expr->getOperator();
+        switch (curOp)
+        {
+        case no_nofold:
+        case no_section:
+        case no_sectioninput:
+        case no_preservemeta:
+        case no_nocombine:
+        case no_forcegraph:
```

Comment:
No, because it is so rare/unused that I don't really mind what happens if it is used.  But it would be more logical to skip it.